diff --git a/pkgs/fiona/default.nix b/pkgs/fiona/default.nix
index 753bcc3..156cb26 100644
--- a/pkgs/fiona/default.nix
+++ b/pkgs/fiona/default.nix
@@ -3,7 +3,7 @@
   buildPythonPackage,
   pythonOlder,
   fetchFromGitHub,
-  cython,
+  cython_3,
   gdal,
   oldest-supported-numpy,
   setuptools,
@@ -35,7 +35,7 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [
-    cython
+    cython_3
     gdal # for gdal-config
     oldest-supported-numpy
     setuptools
