diff --git a/pkgs/pg_tileserv/default.nix b/pkgs/pg_tileserv/default.nix
index 38d59d5..1355efe 100644
--- a/pkgs/pg_tileserv/default.nix
+++ b/pkgs/pg_tileserv/default.nix
@@ -17,12 +17,12 @@ buildGoModule rec {
     # fix default configuration file location
     substituteInPlace \
       main.go \
-      --replace-fail "viper.AddConfigPath(\"/etc\")" "viper.AddConfigPath(\"$out/share/config\")"
+      --replace "viper.AddConfigPath(\"/etc\")" "viper.AddConfigPath(\"$out/share/config\")"
 
     # fix assets location in configuration file
     substituteInPlace \
       config/pg_tileserv.toml.example \
-      --replace-fail "# AssetsPath = \"/usr/share/pg_tileserv/assets\"" "AssetsPath = \"$out/share/assets\""
+      --replace "# AssetsPath = \"/usr/share/pg_tileserv/assets\"" "AssetsPath = \"$out/share/assets\""
   '';
 
   ldflags = [ "-s" "-w" "-X main.programVersion=${version}" ];
