diff --git a/pkgs/postgis/postgis.nix b/pkgs/postgis/postgis.nix
index 6f25390..1a50583 100644
--- a/pkgs/postgis/postgis.nix
+++ b/pkgs/postgis/postgis.nix
@@ -81,7 +81,7 @@ stdenv.mkDerivation rec {
     homepage = "https://postgis.net/";
     changelog = "https://git.osgeo.org/gitea/postgis/postgis/raw/tag/${version}/NEWS";
     license = licenses.gpl2;
-    maintainers = with maintainers; teams.geospatial.members ++ [ marcweber wolfgangwalther ];
+    maintainers = with maintainers; teams.geospatial.members ++ [ marcweber ];
     inherit (postgresql.meta) platforms;
     broken = versionOlder postgresql.version "12";
   };
