diff --git b/pkgs/qgis/unwrapped-ltr.nix a/pkgs/qgis/unwrapped-ltr.nix
index d29561a..0c92606 100644
--- b/pkgs/qgis/unwrapped-ltr.nix
+++ a/pkgs/qgis/unwrapped-ltr.nix
@@ -3,7 +3,7 @@
 , makeWrapper
 , mkDerivation
 , substituteAll
-, wrapGAppsHook3
+, wrapGAppsHook
 , wrapQtAppsHook
 
 , withGrass ? true
@@ -90,7 +90,7 @@ in mkDerivation rec {
 
   nativeBuildInputs = [
     makeWrapper
-    wrapGAppsHook3
+    wrapGAppsHook
     wrapQtAppsHook
 
     bison
diff --git b/pkgs/qgis/unwrapped.nix a/pkgs/qgis/unwrapped.nix
index cb09669..8d7f7d1 100644
--- b/pkgs/qgis/unwrapped.nix
+++ a/pkgs/qgis/unwrapped.nix
@@ -3,7 +3,7 @@
 , makeWrapper
 , mkDerivation
 , substituteAll
-, wrapGAppsHook3
+, wrapGAppsHook
 , wrapQtAppsHook
 
 , withGrass ? true
@@ -90,7 +90,7 @@ in mkDerivation rec {
 
   nativeBuildInputs = [
     makeWrapper
-    wrapGAppsHook3
+    wrapGAppsHook
     wrapQtAppsHook
 
     bison
