diff --git b/pkgs/shapely/default.nix a/pkgs/shapely/default.nix
index 28f2acd..209cdf3 100644
--- b/pkgs/shapely/default.nix
+++ a/pkgs/shapely/default.nix
@@ -5,7 +5,7 @@
 , pytestCheckHook
 , pythonOlder
 
-, cython_0
+, cython
 , geos
 , numpy
 , oldest-supported-numpy
@@ -26,7 +26,7 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [
-    cython_0
+    cython
     geos # for geos-config
     oldest-supported-numpy
     setuptools
