diff --git a/pkgs/shapely/default.nix b/pkgs/shapely/default.nix
index 9819cf0..415317e 100644
--- a/pkgs/shapely/default.nix
+++ b/pkgs/shapely/default.nix
@@ -6,7 +6,7 @@
   pytestCheckHook,
   pythonOlder,
 
-  cython_0,
+  cython,
   geos,
   numpy,
   oldest-supported-numpy,
@@ -27,7 +27,7 @@ buildPythonPackage rec {
   };
 
   nativeBuildInputs = [
-    cython_0
+    cython
     geos # for geos-config
     oldest-supported-numpy
     setuptools
