diff --git a/pkgs/rasterio/default.nix b/pkgs/rasterio/default.nix
index 158394a..5143f3c 100644
--- a/pkgs/rasterio/default.nix
+++ b/pkgs/rasterio/default.nix
@@ -14,7 +14,7 @@
   click,
   click-plugins,
   cligj,
-  cython,
+  cython_3,
   gdal,
   hypothesis,
   ipython,
@@ -55,7 +55,7 @@ buildPythonPackage rec {
   '';
 
   nativeBuildInputs = [
-    cython
+    cython_3
     gdal
     numpy
     setuptools
