diff --git b/pkgs/grass/default.nix a/pkgs/grass/default.nix
index c42984a..9294c33 100644
--- b/pkgs/grass/default.nix
+++ a/pkgs/grass/default.nix
@@ -54,7 +54,7 @@ stdenv.mkDerivation (finalAttrs: {
     libmysqlclient # for `mysql_config`
     netcdf # for `nc-config`
     pkg-config
-  ] ++ (with python3Packages; [ python-dateutil numpy wxpython ]);
+  ] ++ (with python3Packages; [ python-dateutil numpy wxPython_4_2 ]);
 
   buildInputs = [
     blas
