diff --git b/pkgs/pdal/default.nix a/pkgs/pdal/default.nix
index c035460..eb96889 100644
--- b/pkgs/pdal/default.nix
+++ a/pkgs/pdal/default.nix
@@ -10,7 +10,7 @@
 , curl
 , gdal
 , hdf5-cpp
-, laszip
+, LASzip
 , libe57format
 , libgeotiff
 , libtiff
@@ -45,7 +45,7 @@ stdenv.mkDerivation (finalAttrs: {
     curl
     gdal
     hdf5-cpp
-    laszip
+    LASzip
     libgeotiff
     libtiff
     libxml2
@@ -92,7 +92,6 @@ stdenv.mkDerivation (finalAttrs: {
     # This test should be re-enabled once https://github.com/PDAL/PDAL/pull/4411
     # is merged !
     "pdal_io_ogr_writer_test"
-
     # Tests failing due to TileDB library implementation, disabled also
     # by upstream CI.
     # See: https://github.com/PDAL/PDAL/blob/bc46bc77f595add4a6d568a1ff923d7fe20f7e74/.github/workflows/linux.yml#L81
