diff --git a/pkgs/pg_featureserv/default.nix b/pkgs/pg_featureserv/default.nix
index 04a2266..7564bec 100644
--- a/pkgs/pg_featureserv/default.nix
+++ b/pkgs/pg_featureserv/default.nix
@@ -17,12 +17,12 @@ buildGoModule rec {
     # fix default configuration file location
     substituteInPlace \
       internal/conf/config.go \
-      --replace-fail "viper.AddConfigPath(\"/etc\")" "viper.AddConfigPath(\"$out/share/config\")"
+      --replace "viper.AddConfigPath(\"/etc\")" "viper.AddConfigPath(\"$out/share/config\")"
 
     # fix assets location in configuration file
     substituteInPlace \
       config/pg_featureserv.toml.example \
-      --replace-fail "AssetsPath = \"./assets\"" "AssetsPath = \"$out/share/assets\""
+      --replace "AssetsPath = \"./assets\"" "AssetsPath = \"$out/share/assets\""
   '';
 
   ldflags = [ "-s" "-w" "-X github.com/CrunchyData/pg_featureserv/conf.setVersion=${version}" ];
